base-image/assets/prepare_version.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#see man login(1) and ENVIRON(7)
+SHELL=/bin/bash
+USER=travis
+LOGNAME=travis
+LC_MESSAGES=POSIX
+LANG=en_US.UTF-8
+
+CCOMMIT_FILE=/home/travis/$PROJECT_NAME/.current-commit
+[ -f $CCOMMIT_FILE ] && [ $FINAL_VERSION == $(cat $CCOMMIT_FILE) ] && exit 0
+
+echo "The requested version needs to be re-checked out....."
+
+source ~/.rvm/scripts/rvm
```

Comment:
just question: what about make env.sh to initialize `rvm` too? That makes code more readable to have all initialization on one place.